superset/assets/src/explore/AdhocFilter.js

```diff
@@ -45,7 +45,8 @@ export default class AdhocFilter {
       this.clause = adhocFilter.clause;
       this.sqlExpression = null;
```

Comment:
sqlExpression can be set to null here, so you can't predict whether it will be `undefined` or `null`